src/Nordea/Components/MultiSelectDropdown.elm

```diff
@@ -327,6 +387,16 @@ viewSelectItems dropdown index orgOptionGroup =
             )
 
 
+focusInput : InputProperties msg -> Cmd msg
+focusInput inputProperties =
+    Task.attempt (\_ -> inputProperties.noOp) (Browser.focus (makeId inputProperties))
```

Comment:
Having a noop feels a bit cumbersome. I think it is fine to fire onInput with the current input for example. I mean, you did get some input, kinda like how a space + backspace input would work like a noop.

Reply:
so i guess it will be
```
onClick (inputProperties.onInput inputProperties.input (
Task.attempt (\_ -> inputProperties.onInput inputProperties.input Cmd.none) (Browser.focus (makeId inputProperties))))
```

😅 @fredrikloberg